unittests/CMakeLists.txt

```diff
@@ -19,10 +19,11 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/include/config.hpp.in ${CMAKE_CURRENT
 file(GLOB UNIT_TESTS "*.cpp")
 
 add_executable( unit_test ${UNIT_TESTS} ${WASM_UNIT_TESTS} )
-target_link_libraries( unit_test eosio_chain chainbase eosio_testing eos_utilities abi_generator fc ${PLATFORM_SPECIFIC_LIBS} )
+target_link_libraries( unit_test eosio_chain chainbase chain_plugin eosio_testing eos_utilities abi_generator fc ${PLATFORM_SPECIFIC_LIBS} )
```

Comment:
Dan explicitly didn't want any plugins linked to unittests. Move this test to tests/plugin_test. See tests/CMakeLists.txt. tests/wallet_tests.cpp is an example.